### DISCUSSION TITLE
2021-08-23 django 시작 / github 탐험기

### DISCUSSION THREAD

**Discussion by devpla** [hing9u/django-tutorial#12] [Daily]
## 1. Development
- 장고 시작 ( #2)
- 어플리케이션 추가 ( #4)

<br>

## 2. Docs
- README 작성 ( #1)
- TIL docs 작성 ( #6)

<br>

### 1️⃣ Issue
- 템플릿 및 라벨 추가
![image](https://user-images.githubusercontent.com/87457066/130477851-40b84160-ed41-4d54-9d0e-ad007c508efd.png)
- 라벨에 진심이 되었더니 마음에 들어서 Issue 및 PR title에 타입 삭제함 (중복되는 정보이므로)
- 오늘 한 작업들 모두 Issue로 열고 완료할 때마다 하나씩 닫았다.
- 내일 할 일 미리 open해놨는데 엄청 많다.

<br>

### 2️⃣ Project
- Issue 및 PR과 연동 신기했음.
- 칸반보드 👍

<br>

### 3️⃣ Wiki
- docs를 여기에 쭉 정리하는 것도 좋아보임.
- https://github.com/hing9u/django-tutorial/wiki

<br>

### 4️⃣ Discussions
- commit보다 댓글 남기기 좋은 것 같은데?

<br>

**Comment by chaselover** [comment]
리뷰 남기기 좋은 날이네요

**Reply by devpla** [reply]
오늘은 가성비가 별로인 것 같네요.. 🥺
